### DISCUSSION TITLE
hook and filter feature like WordPress

### DISCUSSION THREAD

**Discussion by mprince2k18** [laravel/framework#36494] [Ideas]
We need a hook and filter feature like WordPress. for extending the project as much big as we want.
I made a custom hook system that is pretty much hardcoded of PHP. This is so overwhelmed.

**Comment by rodrigopedra** [comment]
Aren't events a better approach to this?

https://laravel.com/docs/8.x/events

**Comment by crynobone** [comment]
Hey there,

We're closing this issue because it's inactive, already solved, old, or not relevant anymore. Feel free to open up a new issue if you're still experiencing this problem.
